crop_classification/dl/config.py

```diff
@@ -8,3 +8,12 @@
     "coffee": CropClassifications.beverage_spice,
     "almond": CropClassifications.fruits_nuts,
 }
+
+# LSTM model configurations
+HIDDEN_VECTOR_SIZE = 128
+NUM_CLASSIFICATION_LAYERS = 2
+CLASSIFIER_DROPOUT = 0.2
+CLASSIFIER_BASE_LAYERS = 1
+
+ENCODER_VECTOR_SIZES = [32, 64, 128]
+ENCODER_DROPOUT = 0.2
```

Comment:
I moved the config here because it's all relevant to `crop_classification/dl/timl.py` and this makes it easy to import. What do you think about this? Do you think `crop_classification/dl/timl.py` should reach into `crop_classification/config.py` instead? Should it touch the config at all?

Reply:
yea this is fine for me. I can see the benefits of both ways, but I like what this allows (e.g. the default args you did)